package.json

```diff
@@ -12,7 +12,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "^3.4.3"
```

Comment:
any reason why we need to update the version here? just curious.

Reply:
I ran `npm install` and it showed some warnings so I ran `npm audit fix` and that updated this version which I guess will reduce any vulnerabilities we may find in the future. Also, adding `^` helps use any version greater than or equal to the mentioned one

Reply:
I see. In that case, can you please add to the PR description under `Type of Change` that extra info: "update react-scripts dependency from version 3.4.1 to 3.4.3" follows by instruction for other contributors to update this dependency locally

Reply:
Done